tests/test_tokenization_roformer.py

```diff
@@ -22,21 +22,21 @@
 from .test_tokenization_common import TokenizerTesterMixin
 
 
-def is_rjieba_available():
-    return importlib.util.find_spec("rjieba") is not None
+def is_jieba_available():
+    return importlib.util.find_spec("jieba") is not None
 
 
-def require_rjieba(test_case):
+def require_jieba(test_case):
     """
     Decorator marking a test that requires Jieba. These tests are skipped when Jieba isn't installed.
     """
-    if not is_rjieba_available():
-        return unittest.skip("test requires rjieba")(test_case)
+    if not is_jieba_available():
+        return unittest.skip("test requires jieba")(test_case)
     else:
         return test_case
```

Comment:
`is_jieba_available()` should not be defined within the test file. Rather, it should be defined in the `file_utils.py` file, as can be seen [here](https://github.com/huggingface/transformers/blob/f866425898ee47aea399c949e82ad6b783c42f47/src/transformers/file_utils.py).

`require_jieba` should be defined in `testing_utils.py` instead of here. You can check how this was done for the timm library as an example [here](https://github.com/huggingface/transformers/blob/0d97ba8a9846b4856f8920bf0c955dbce91e8b4f/src/transformers/testing_utils.py#L233).

Reply:
Thanks, I have changed this :) .